core/app/mailers/spree/reimbursement_mailer.rb

```diff
@@ -1,11 +1,11 @@
 module Spree
   class ReimbursementMailer < BaseMailer
-      def reimbursement_email(reimbursement, resend = false)
-        @reimbursement = reimbursement.respond_to?(:id) ? reimbursement : Spree::Reimbursement.find(reimbursement)
-        store = @reimbursement.order.store
-        subject = (resend ? "[#{Spree.t(:resend).upcase}] " : '')
-        subject += "#{store.name} #{Spree.t('reimbursement_mailer.reimbursement_email.subject')} ##{@reimbursement.order.number}"
-        mail(to: @reimbursement.order.email, from: from_address(store), subject: subject)
-      end
+    def reimbursement_email(reimbursement, resend = false)
+      @reimbursement = reimbursement.respond_to?(:id) ? reimbursement : Spree::Reimbursement.find(reimbursement)
+      store = @reimbursement.order.store
+      subject = (resend ? "[#{Spree.t(:resend).upcase}] " : '')
+      subject += "#{store.name} #{Spree.t('reimbursement_mailer.reimbursement_email.subject')} ##{@reimbursement.order.number}"
+      mail(to: @reimbursement.order.email, from: from_address(store), subject: subject)
+    end
```

Comment:
Can you PR this in a separate PR that just fixes the indentation? I'm good with the change, just don't want to mix it up with some of the other stuff in this PR.